### PR TITLE
Update procedure for getting a dataset into Presto.

### PR DESCRIPTION
**Pull request by mreid-moz**
Remove `:robotblake` as the default assignee and flag it for triage by the Data Ops team. This is per `:melissa`'s request.

### CODE REVIEW

File: cookbooks/create_a_dataset.md [hunk 1]
```diff
@@ -7,8 +7,8 @@
...
+3. Using [this template](https://bugzilla.mozilla.org/enter_bug.cgi?bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&cf_fx_iteration=---&cf_fx_points=1&comment=Location%20of%20the%20dataset%3A%20%0D%0ADesired%20dataset%20name%3A&component=Datasets%3A%20General&contenttypemethod=autodetect&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-607=X&flag_type-800=X&flag_type-803=X&flag_type-916=X&form_name=enter_bug&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Linux&priority=P1&product=Data%20Platform%20and%20Tools&rep_platform=x86_64&short_desc=Add%20dataset%20to%20Presto&target_milestone=---&version=unspecified),
    open a bug to load the dataset in Presto with the following attributes:
-   * Assigned to `:robotblake`
+   * Add whiteboard tag `[DataOps]`
    * Title: "Add Dataset to Presto"
-   * Content: Location of the dataset and the desired table name
+   * Content: Location of the dataset in S3 (from step 2 above) and the desired table name in Presto
```
Comment (fbertsch) [inline cookbooks/create_a_dataset.md]: nit: it's not just for Presto, I think `desired table name when querying` could work if you think it needs more context.

Comment (mreid-moz) [reply cookbooks/create_a_dataset.md]: Should the bug title change too then? Maybe `"Add Dataset to metastore"`?

Comment (fbertsch) [reply cookbooks/create_a_dataset.md]: Should we maybe call it `"Publish dataset"` or similar? I don't want people to have to learn what a metastore is, and it does more than just make it available in `Re:dash`. Naming is hard.

Comment (mreid-moz) [reply cookbooks/create_a_dataset.md]: `"Publish dataset"` wfm!

Comment (mreid-moz) [reply cookbooks/create_a_dataset.md]: @fbertsch RFAL

**Pull request merged**